requirements.txt

```diff
@@ -1,34 +1,35 @@
+alembic==1.6.5
 APScheduler==3.8.0
+bcrypt~=3.2.0
+boto3~=1.18.14
+click==7.1.2
+cryptography~=3.4.8
+dask==2021.10.0
+email-validator
+emails
+fastapi-caching[redis]
+fastapi-pagination[sqlalchemy]~= 0.8.3
 fastapi[all]==0.68.1
-sqlalchemy==1.4.14
-alembic==1.6.5
 fidesctl==0.9.8.4
-python-jose[cryptography]==3.3.0
+multidimensional_urlencode==0.0.4
+pandas==1.3.3
 passlib[bcrypt]==1.7.2
 psycopg2-binary==2.9.1
-email-validator
-emails
-fastapi-caching[redis]
-sqlalchemy-stubs==0.4
-SQLAlchemy-Utils==0.37.8
-redis==3.5.3
 pydantic~=1.8.2
-starlette~=0.14.2
-python-dotenv~=0.19.0
-bcrypt~=3.2.0
-Unidecode==1.2.0
-uvicorn~=0.13.4
 pydash==5.0.2
-boto3~=1.18.14
-cryptography~=3.4.8
-fastapi-pagination[sqlalchemy]~= 0.8.3
-dask==2021.10.0
-requests~=2.25.0
 pymongo==3.12.0
-pandas==1.3.3
-click==7.1.2
 PyMySQL==1.0.2
-sqlalchemy-redshift==0.8.8
+python-dotenv~=0.19.0
+python-jose[cryptography]==3.3.0
+redis==3.5.3
+requests~=2.25.0
 snowflake-sqlalchemy==1.3.2
 sqlalchemy-bigquery==1.3.0
-multidimensional_urlencode==0.0.4
\ No newline at end of file
+sqlalchemy-redshift==0.8.8
+sqlalchemy-stubs==0.4
+SQLAlchemy-Utils==0.37.8
+sqlalchemy==1.4.14
+starlette~=0.14.2
+Unidecode==1.2.0
+uvicorn~=0.13.4
+versioneer==0.19
```

Comment:
This line is the only addition in this file. The large diff is due to the alphabetization change, which will pay for itself in saved time (after about 200 years).